### ISSUE TITLE
Discrepancy in MatMul Slides vs. Demo

### ISSUE THREAD

**Issue by werrah** [uwhpsc-2016/lectures#9]
In the Lecture 8 slides, the final line in MatMul is C[ i \* N + j ] += A[ i \* N +k ] \* B[ k \* N + j ]. That first and last N should be K, correct? In the demo, that line is changed to C[ K \* i + j ] += A[ N \* i +k ] \* B[ K \* k + j ]. When I was working it out by hand so it would make sense in my head, the demo code made sense, but the lecture slides didn't. Is that a mistake in the slides, or is there something I'm missing that would explain whey the two are different?

**Comment by cswiercz** [comment]
You are absolutely correct. There was a typo (two, in fact) in the lecture slides. I will upload a correction.

**Comment by cswiercz** [comment]
Corrections uploaded. I will notify the class along with the quiz reminder.
